### PR TITLE
[Surrey] Disallow email addresses in problem title/detail

### PR DESCRIPTION
**Pull request by davea**
Fixes https://github.com/mysociety/societyworks/issues/4565

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/Surrey.pm [hunk 1]
```diff
@@ -86,6 +86,24 @@ sub reopening_disallowed { 1 }
...
+
+=cut
+
+sub report_validation {
+    my ($self, $report, $errors) = @_;
+
+    my $regex = Utils::email_regex;
+
+    if ($report->detail =~ /$regex/ || $report->title =~ /$regex/) {
+        $errors->{detail} = 'Please remove any email addresses and other personal information from report';
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Surrey.pm:101]: "information from report" needs rewording

**Review by dracos — CHANGES_REQUESTED** [review]
One wording change is all

File: perllib/FixMyStreet/Cobrand/Surrey.pm (hunk 1 above)
Comment (davea) [reply perllib/FixMyStreet/Cobrand/Surrey.pm:101]: > "information from report" needs rewording

Would "information from your report" be OK? Or do you have something else in mind?

Comment (dracos) [reply perllib/FixMyStreet/Cobrand/Surrey.pm:101]: Yeah, that's fine, it just wasn't English as is :)

**Review by dracos — APPROVED** [review]

**Pull request merged**